lib/babel.js

```diff
@@ -16,6 +16,11 @@ if (debug.enabled) {
 // Bind globals first, before anything has a chance to interfere.
 var globals = require('./globals');
 
+var resolveCwd = require('resolve-cwd');
+(opts.require || []).forEach(function (moduleId) {
```

Comment:
I think `opts.require` is already guaranteed to be an array, so this check is moot.

https://github.com/sindresorhus/ava/pull/296/files#diff-2cce40143051e25f811b56c79d619bf5R73